packages/esm-patient-vitals-app/src/common/helpers.ts

```diff
@@ -33,24 +33,41 @@ export function assessValue(value: number | undefined, range?: ObsReferenceRange
   return 'normal';
 }
 
+export function interpretFhirInterpretation(interpretation: FHIRInterpretation): ObservationInterpretation {
+  switch (interpretation) {
+    case 'Critically Low':
+      return 'critically_low';
+    case 'Critically High':
+      return 'critically_high';
+    case 'High':
+      return 'high';
+    case 'Low':
+      return 'low';
+    case 'Normal':
+      return 'normal';
+  }
+}
+
 export function interpretBloodPressure(
   systolic: number | undefined,
   diastolic: number | undefined,
   concepts: { systolicBloodPressureUuid?: string; diastolicBloodPressureUuid?: string } | undefined,
   conceptMetadata: Array<ConceptMetadata> | undefined,
+  systolicInterpretation?: ObservationInterpretation,
+  diastolicInterpretation?: ObservationInterpretation,
 ): ObservationInterpretation {
   if (!conceptMetadata) {
     return 'normal';
   }
 
-  const systolicAssessment = assessValue(
-    systolic,
-    getReferenceRangesForConcept(concepts?.systolicBloodPressureUuid, conceptMetadata),
-  );
+  const systolicAssessment =
+    systolicInterpretation ??
+    assessValue(systolic, getReferenceRangesForConcept(concepts?.systolicBloodPressureUuid, conceptMetadata));
```

Comment:
```suggestion
  const systolicAssessment =
    systolicInterpretation ??
    (concepts?.systolicBloodPressureUuid
      ? assessValue(systolic, getReferenceRangesForConcept(concepts.systolicBloodPressureUuid, conceptMetadata))
      : 'normal');
```

We should change this to align with the logic in `diastolicAssessment` for consistency.